cmd/urunc/main.go

```diff
@@ -50,19 +53,14 @@ your host. Providing the bundle directory using "-b" is optional. The default
 value for "bundle" is the current directory.`
 )
 
-var Log = l.BaseLogEntry()
 var version string
 
 func main() {
-	Log.WithField("args", os.Args).Info("urunc INVOKED")
-	// to previous line
-
+	root := "/run/urunc"
```

Comment:
we need to verify that mount options match what we want (eg. if we need the exec flag enabled). If by default they are not, we should do what kata does (remount/bind mount etc.)

Additionally, we should add a comment about the fact that we fill up /run with unikernel binaries (unless this is resolved so please just ignore this)

Reply:
Usually, when executing a unikernel container, we have to deal with 2 different container directories. 

The first one is this root directory (`/run/containerd/runc/<NAMESPACE>` or the default value `/run/urunc`) usually passed down to us from the `containerd-shim`. Inside this directory, resides a child directory for each container (`/run/containerd/runc/<NAMESPACE>/<CONTAINER_ID>`). We use that container-specific directory to store our `state.json` file.

The second one (which is usually the `cwd` of the `urunc` process) is the bundle path and (in most cases) is `/run/containerd/io.containerd.v2.task/<NAMESPACE>/<CONTAINER_ID>`. This is where our rootfs, init.pid etc are stored.

So, as far as I understand, the root directory is not mounted.

---

The comment about filling up /run is definitely needed, I will add it to the relevant function.